.github/workflows/e2e-nextjs.yml

```diff
@@ -0,0 +1,83 @@
+name: Next.js Integration test
+
+on:
+  pull_request:
+    branches: [main]
+    # Run on release PRs only
+    paths: ['CHANGELOG.md']
+
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        bundler: ['default', 'esbuild']
+    steps:
+      - name: Checking out zip-it-and-ship-it
+        uses: actions/checkout@v2
+        with:
+          path: zip-it-and-ship-it
+      - name: Checking out netlify-plugin-nextjs
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/netlify-plugin-nextjs
+          path: netlify-plugin-nextjs
+      - name: Checking out netlify-cli
+        uses: actions/checkout@v2
+        with:
+          repository: netlify/cli
+          path: netlify-cli
+      - name: Installing Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: Installing netlify-cli
```

Comment:
The CLI is already a local dependency of the plugin:
https://github.com/netlify/netlify-plugin-nextjs/blob/1c06d1323312bf7b4ff7f4c228978b96a73f2917/package.json#L90
so we might be able to save some time and skip this part (and the checkout).

Reply:
Oh, I didn't know that. It's definitely worth exploring.